Calculator/Calculator/Model/ExpressionParser.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  ExpressionParser.swift
+//  Calculator
+//
+//  Created by Hmeg on 2023/06/06.
+//
+
+enum ExpressionParser {
+    static func parse(from input: String) -> Formula {
+        let operandQueue = CalculatorItemQueue<Double>()
+        let operatorQueue = CalculatorItemQueue<Operator>()
+        
+        let components = componentsByOperators(from: input)
+        
+        components.compactMap { Double($0) }.forEach { operandQueue.enqueue(item: $0)}
+        
+        let operatorComponents = components.filter { Double($0) == nil }
+        operatorComponents.compactMap { Operator(rawValue: Character($0))}
+            .forEach{(operatorQueue.enqueue(item:$0))}
```

Comment:
```suggestion
        components
            .filter { Double($0) == nil }
            .compactMap { Operator(rawValue: Character($0))}
            .forEach{(operatorQueue.enqueue(item:$0))}
```

이렇게 수정해볼 수 있을 것 같아요!